.cargo/config.toml

```diff
@@ -30,7 +30,8 @@ rustflags = [
   "-Wclippy::same_name_method",
 
   "-Aclippy::default_constructed_unit_structs",
-  "-C", "symbol-mangling-version=v0"
+  "-C", "symbol-mangling-version=v0",
+  "--cfg", "tokio_unstable"
```

Comment:
What's this flag all about? Is it intentional?

Reply:
yes, it's required by tokio-console https://github.com/tokio-rs/console#:~:text=in%20order%20to%20collect%20task%20data%20from%20Tokio%2C%20the%20tokio_unstable%20cfg%20must%20be%20enabled.%20for%20example%2C%20you%20could%20build%20your%20project%20with

Reply:
deno also use this flag https://github.com/denoland/deno/blob/main/.cargo/config.toml#L37, so it seems safe to use it